### DISCUSSION TITLE
My Student Pack Application wont get accepted

### DISCUSSION THREAD

**Discussion by dablincx** [community/community#41533] [GitHub Education]
Hi,
I tried to apply for the Student Developer Pack and I used my email provided by my school and I uploaded my school ID but I'm not getting accepted because of the error below. I added the missing information (Full name and Bio) to my profile but it still wont finish processing my application.

The Error:
![huh](https://user-images.githubusercontent.com/80812233/207103012-60618f45-d80e-47ea-9684-16ab874d9fb4.png)

Thanks for any help :)

**Comment by Kel-VinTech** [comment]
Go to your user profile add your full name and a little bio just as it requested and make sure the school email you're using is added to your GitHub email on your profile

**Reply by dablincx** [reply]
my school email is linked to my github account and verified
i added my full name to my profile and a bio

its still not working

**Comment by samus-aran** [comment]
Hi @DaBlincx  👋
We'd love to help! 

As you can imagine GitHub processes a number of applications - we do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification looks at your GitHub username, email and profile name (and other aspects). Do these match? If they do I would suggest to re-apply so that the change has been picked up.
